packages/manager/modules/pci/src/projects/project/private-networks/private-networks.controller.js

```diff
@@ -0,0 +1,59 @@
+import get from 'lodash/get';
+import { VRACK_OPERATION_COMPLETED_STATUS } from './private-networks.constants';
```

Comment:
Separate imports from `node_modules` resolution and for local one would be great.